### PR TITLE
Implement new notifications for chats

### PR DESCRIPTION
**Pull request by aapeliv**
Implements the new notification system for conversations. The actual notifications are generated in the background in case of a huge group chat so that the system is still responsive.

**Backend checklist**
- [x] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [x] Run the backend locally and it works
- [x] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

### CODE REVIEW

File: app/backend/src/couchers/jobs/handlers.py [hunk 1]
```diff
@@ -57,6 +58,54 @@ def process_purge_login_tokens(payload):
...
+
+        notify_args = {
+            "topic": "chat",
+            "key": str(message.conversation_id),
+            "action": "message",
+            "icon": "message",
+            "title": f"{message.author.name} sent a message in {group_chat.title}.",
+            "content": message.text,
+            "link": urls.chat_link(chat_id=message.conversation_id),
+        }
```
Comment (kalvdans) [inline app/backend/src/couchers/jobs/handlers.py]: Avoid the intermediary dict and pass the parameters in the call directly.

Comment (aapeliv) [reply app/backend/src/couchers/jobs/handlers.py]: Fixed

**Review by kalvdans — APPROVED** [review]
LGTM! Thanks for fixing my complaint

**Comment by aapeliv** [comment]
Merged in 528d311c69db2f11898d6d337ac6717beced8737

**Pull request closed without merging**